src/docker-compose/docker-compose-wavefront.yml

```diff
@@ -14,6 +14,9 @@ services:
       - MANAGEMENT_WAVEFRONT_METRICS_EXPORT_APITOKEN=${WAVEFRONT_KEY:?WAVEFRONT_KEY is not set!}
       - MANAGEMENT_WAVEFRONT_METRICS_EXPORT_URI=${WAVEFRONT_URI:-https://vmware.wavefront.com}
       - MANAGEMENT_WAVEFRONT_METRICS_EXPORT_SOURCE=${WAVEFRONT_SOURCE:-scdf-docker-compose}
+      - SPRING_CLOUD_DATAFLOW_APPLICATIONPROPERTIES_STREAM_MANAGEMENT_TRACING_ENABLED=true
```

Comment:
There is some discussion about the usage of this property [here](https://github.com/spring-projects/spring-boot/issues/38625). While I see the auto configuration is enabled by default, the usage of this property is still not recommended and this property is even removed in the later SB 3.x versions.

Reply:
The property is not intended to enable/disable tracing, just the exporting to the reporters. 
See [here](https://github.com/spring-projects/spring-boot/blob/f024c193e4ae5c4263077d8fada4959639231248/spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/tracing/OnEnabledTracingCondition.java#L36-L37). THere is a global and a reporter specific variant. 

This one is actually redundant since we are setting 
```
SPRING_CLOUD_DATAFLOW_APPLICATIONPROPERTIES_STREAM_MANAGEMENT_WAVEFRONT_TRACING_EXPORT_ENABLED=true
```
below. 

I will remove it from the stream apps though and here as well in a subsequent PR as it does not add value.

Thanks for the good eye!

Reply:
Created https://github.com/spring-cloud/spring-cloud-dataflow/issues/5928 to follow up